routes/web.php

```diff
@@ -55,6 +55,8 @@ function () {
 
                 Route::get("language", ["as" => "language.edit", "uses" => "LanguageSettingController@edit"]);
                 Route::put("language", ["as" => "language.update", "uses" => "LanguageSettingController@update"]);
+                Route::post("language/store", ["as" => "language.store", "uses" => "LanguageSettingController@store"]);
```

Comment:
For this section, you can name your routes like:

language/item  (get)  as language.item.create
language/item  (post) as language.item.store